mantidproject/mantid
# Automatically calculate ConvertToDiffractionMD extents

This changes the extents property on `ConvertToDiffractionMD2` to use `ConvertToMDMinMaxLocal` to calculate extents for the output MD workspace.

**To test:**
 - Check the unit & system tests are passing
 - Try running ConvertToDiffractionMD both with and entering values for the extents.
    - Check these work calculated extents are correct for the instrument attached to the workspace
 - Code review

Fixes #18395 .

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
